Tests/FlexLayoutTests/FlexLayoutTests.swift

```diff
@@ -26,5 +26,18 @@ final class FlexLayoutTests: XCTestCase {
 
         XCTAssertNil(weakView, "Creation of flex should not lead to retain cycle")
     }
+  
+    
+    func testRemoveViewDynamically() {
+        let rootFlexContainer = UIView()
+        rootFlexContainer.flex.addItem(UIView())
+        rootFlexContainer.flex.define { _ in }
+        rootFlexContainer.flex.layout()
+        
+        rootFlexContainer.subviews.forEach { $0.removeFromSuperview() }
+        rootFlexContainer.flex.markDirty()
+        rootFlexContainer.flex.layout()
 
+        XCTAssertTrue(rootFlexContainer.subviews.isEmpty)
```

Comment:
Calling markDirty() on rootFlexContainer is unnecessary, but it's an area where users can easily make mistakes.

In the code before this PR, it bypasses the early return condition and causes an error.